.storybook/preview.js

```diff
@@ -64,6 +64,7 @@ export const parameters = {
     },
     docs: {
         // Customize docs page layout (in order to rename 'Stories' section)
+        /* eslint-disable-next-line react/display-name */
```

Comment:
We could also disable this rule globally for all `*.stories.js` and for everything in `.storybook` with an [override](https://eslint.org/docs/user-guide/configuring/configuration-files#how-do-overrides-work).

Reply:
Shall we perhaps leave that out-of-scope for now?
If we change this later, after discussing, we will get a warning/error about the superfluous comment anyway.